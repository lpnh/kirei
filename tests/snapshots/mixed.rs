#[test]
fn for_list() {
    format_and_snapshot!("mixed/for_list.html");
}

#[test]
fn nested_blocks() {
    format_and_snapshot!("mixed/nested_blocks.html");
}

#[test]
fn block_surrounded_by_html() {
    format_and_snapshot!("mixed/block_surrounded_by_html.html");
}

#[test]
fn nesting_macros() {
    format_and_snapshot!("mixed/nesting_macros.html");
}

#[test]
fn base_template() {
    format_and_snapshot!("mixed/base_template.html");
}

#[test]
fn head() {
    format_and_snapshot!("mixed/head.html");
}

#[test]
fn full_blown() {
    format_and_snapshot!("mixed/full_blown.html");
}

#[test]
fn simple_template() {
    format_and_snapshot!("mixed/simple_template.html");
}
