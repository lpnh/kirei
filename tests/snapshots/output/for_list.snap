---
source: tests/snapshots/mixed.rs
description: "<h1>Users</h1><ul>{% for user in users %}<li>{{ user.name }}</li>{% endfor %}</ul>\n"
---
<h1>Users</h1>
<ul>
    {% for user in users %}<li>{{ user.name }}</li>{% endfor %}
</ul>
